source/FilterableLogMonitor.js

```diff
@@ -102,20 +113,26 @@ export default class FilterableLogMonitor extends Component {
           color: theme.base07,
           fontSize: 14,
           minWidth: 200
-        }}
-      >
+        }}>
         <ActionFilter
           actionFilterText={actionFilterText}
           dispatch={dispatch}
           theme={theme}
         />
         <div
+          ref={(node) => { this.wrapperNode = node }}
           style={{
             flex: '1',
             overflowY: 'scroll'
           }}
-        >
-          {filterableStates}
+          onScroll={() => {
+            if (this.wrapperNode.scrollTop + this.wrapperNode.offsetHeight >= this.containerNode.offsetHeight) {
+              this.isAutoScrollEnabled = true
+              return
+            }
+            this.isAutoScrollEnabled = false
```

Comment:
nit: This could be reduced to:

```js
this.isAutoScrollEnabled =
  this.wrapperNode.scrollTop + this.wrapperNode.offsetHeight >=
  this.containerNode.offsetHeight;
```